types/artifacts.d.ts

```diff
@@ -213,6 +215,14 @@ declare global {
         requestId: string | null
       }
 
+      export type SourceMap = {
+        url: string
+        map: import('source-map').RawSourceMap
+      } | {
+        url: string
+        error: string
```

Comment:
WDYT about `errorMessage` to match our other usage and the fact that it's a string not `Error`?